### ISSUE TITLE
MA0124 does not recognize nullable types correctly

### ISSUE THREAD

**Issue by FreeYourMinds** [meziantou/Meziantou.Analyzer#489]
**Context**
Rule identifier: _MA0124 / MA0125_
Target Framework: _.NET 6.0 / .NET 7.0_
Language version: _10.0 / 11.0_

**Describe the bug**

MA0124 does not recognize nullable types correctly:
![grafik](https://user-images.githubusercontent.com/36032876/229232290-a906f629-c9b2-4d84-acb1-5514af118825.png)

Configuration for `ArticleId` in MA0124 looks like

```
ArticleId;System.Guid;T:System.Nullable{System.Guid}
```

Trying to add `;T:System.Guid?` to the configuration results in _MA0125_ firing:

> [MA0125] The type 'T:System.Guid?' does not match any symbol of the compilation

**To Reproduce**

Configure a parameter to be _System.Guid?_ and setup MA0124 accrodingly

````c#
Guid? articleIdToSingularize = null;
this.logger.LogInformation("ArticleId to singularize: {ArticleId}", articleIdToSingularize);
````

**Additional context**

_None_

**Comment by meziantou** [comment]
Thanks for reporting the issue! The next version that includes the fix should be available in a few minutes.

Note that the txt file syntax will be different for generic types:

````
# Before
ArticleId;System.Guid;T:System.Nullable{System.Guid}

# After
ArticleId;System.Guid;System.Nullable{System.Guid}
````

**Comment by FreeYourMinds** [comment]
Works perfectly. 

Thank you for this incredibly quick fixing 👍
